### ISSUE TITLE
Bug with long feedback messages

### ISSUE THREAD

**Issue by Xyene** [DMOJ/site#245]
```
Error in packet handling (Judge-side)
Traceback (most recent call last):
  File "/code/site/judge/bridge/judgehandler.py", line 168, in packet
    handler(data)
  File "/code/site/judge/bridge/judgecallback.py", line 293, in on_test_case
    test_case.save()
  File "/code/pypy-virtenv/site-packages/django/db/models/base.py", line 734, in save
    force_update=force_update, update_fields=update_fields)
  File "/code/pypy-virtenv/site-packages/django/db/models/base.py", line 762, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/code/pypy-virtenv/site-packages/django/db/models/base.py", line 846, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/code/pypy-virtenv/site-packages/django/db/models/base.py", line 885, in _do_insert
    using=using, raw=raw)
  File "/code/pypy-virtenv/site-packages/django/db/models/manager.py", line 127, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/code/pypy-virtenv/site-packages/django/db/models/query.py", line 920, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/code/pypy-virtenv/site-packages/django/db/models/sql/compiler.py", line 974, in execute_sql
    cursor.execute(sql, params)
  File "/code/pypy-virtenv/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/code/pypy-virtenv/site-packages/django/db/utils.py", line 97, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/code/pypy-virtenv/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/code/pypy-virtenv/site-packages/django/db/backends/mysql/base.py", line 124, in execute
    return self.cursor.execute(query, args)
  File "/code/pypy-virtenv/site-packages/MySQLdb/cursors.py", line 220, in execute
    self.errorhandler(self, exc, value)
  File "/code/pypy-virtenv/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorvalue
DataError: (1406, "Data too long for column 'feedback' at row 1")

Request repr(): unavailable
```

Two bugs in one! First, the bridge crashes, and secondly, it's reported as being a "Judge-side" error.

**Comment by quantum5** [comment]
I find your lack of basis disturbing. By "Judge-side", it means the part of the bridge facing the judges, instead of that which faces the site.

**Comment by quantum5** [comment]
Isn't this fixed in c768dfe3fd50ed7d86e13ac10090e63892a9035f?
